cluster/aws/util.sh

```diff
@@ -411,9 +411,9 @@ function create-security-group {
 
   local sgid=$(get_security_group_id "${name}")
   if [[ -z "$sgid" ]]; then
-	  echo "Creating security group ${name}."
-	  sgid=$($AWS_CMD create-security-group --group-name "${name}" --description "${description}" --vpc-id "${VPC_ID}" --query GroupId)
-	  add-tag $sgid KubernetesCluster ${CLUSTER_ID}
+          echo "Creating security group ${name}."
+          sgid=$($AWS_CMD create-security-group --group-name "${name}" --description "${description}" --vpc-id "${VPC_ID}" --query GroupId)
+          add-tag $sgid KubernetesCluster ${CLUSTER_ID}
```

Comment:
This is normalizing some previously-merged lines from tab to 2-space (My fault from previous PRs). I ran an emacs macro over the relevant files, so they should all be uniform after this PR.